common/lfdemod.c

```diff
@@ -1067,6 +1089,23 @@ void psk1TOpsk2(uint8_t *BitStream, size_t size)
 	return;
 }
 
+// by marshmellow
+// convert psk2 demod to psk1 demod
+// from only transition waves are 1s to phase shifts change bit
+void psk2TOpsk1(uint8_t *BitStream, size_t size)
```

Comment:
Heh, I knew something was lost when my PC crashed the other day...  It is missing a couple lines....  I'll fix.